atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -398,11 +394,11 @@ private static LockAndTimestampServices createRawEmbeddedServices(
         return AtlasDbHttpClients.createProxyWithFailover(sslSocketFactory, uris, serviceClass, userAgent);
     }
 
-    private static final class ServiceCreator<T> implements Function<ServerListConfig, T> {
+    public static final class ServiceCreator<T> implements Function<ServerListConfig, T> {
```

Comment:
Maybe for this we're looking for the minimal change, but now that this is public would probably be valuable to have as a separate class.